aws/opsworks_layers.go

```diff
@@ -643,3 +764,83 @@ func (lt *opsworksLayerType) SetVolumeConfigurations(d *schema.ResourceData, v [
 
 	d.Set("ebs_volume", newValue)
 }
+
+func (lt *opsworksLayerType) Autoscaling(d *schema.ResourceData) *opsworks.SetLoadBasedAutoScalingInput {
+	return &opsworks.SetLoadBasedAutoScalingInput{
+		Enable:  aws.Bool(d.Get("autoscaling").(bool)),
+		LayerId: aws.String(d.Id()),
+		DownScaling: &opsworks.AutoScalingThresholds{
+			Alarms:             expandStringList(d.Get("downscaling_alarms").([]interface{})),
+			CpuThreshold:       aws.Float64(float64(d.Get("downscaling_cpu_threshold").(float64))),
```

Comment:
FYI, these `float64()` of `float64` types are triggering `make lint` failures in CI since they are unnecessary:

```
$ make lint
==> Checking source code against linters...
aws/opsworks_layers.go:1::warning: file is not gofmted with -s (gofmt)
aws/opsworks_layers.go:774:43:warning: unnecessary conversion (unconvert)
aws/opsworks_layers.go:777:43:warning: unnecessary conversion (unconvert)
aws/opsworks_layers.go:778:43:warning: unnecessary conversion (unconvert)
aws/opsworks_layers.go:783:43:warning: unnecessary conversion (unconvert)
aws/opsworks_layers.go:786:43:warning: unnecessary conversion (unconvert)
aws/opsworks_layers.go:787:43:warning: unnecessary conversion (unconvert)
```

Reply:
Oh! Thank you. I will check it tomorrow.